diff --git a/node_modules/react-native-raw-bottom-sheet/src/index.js b/node_modules/react-native-raw-bottom-sheet/src/index.js
index adc5ee3..db4aa81 100644
--- a/node_modules/react-native-raw-bottom-sheet/src/index.js
+++ b/node_modules/react-native-raw-bottom-sheet/src/index.js
@@ -38,11 +38,13 @@ class RBSheet extends Component {
       this.setState({ modalVisible: visible });
       if (typeof onOpen === "function") onOpen(props);
       Animated.timing(animatedHeight, {
+        useNativeDriver: false,
         toValue: height,
         duration
       }).start();
     } else {
       Animated.timing(animatedHeight, {
+        useNativeDriver: false,
         toValue: minClosingHeight,
         duration
       }).start(() => {
@@ -64,14 +66,14 @@ class RBSheet extends Component {
       onStartShouldSetPanResponder: () => closeOnDragDown,
       onPanResponderMove: (e, gestureState) => {
         if (gestureState.dy > 0) {
-          Animated.event([null, { dy: pan.y }])(e, gestureState);
+          Animated.event([null, { dy: pan.y }], { useNativeDriver: false })(e, gestureState);
         }
       },
       onPanResponderRelease: (e, gestureState) => {
         if (height / 4 - gestureState.dy < 0) {
           this.setModalVisible(false);
         } else {
-          Animated.spring(pan, { toValue: { x: 0, y: 0 } }).start();
+          Animated.spring(pan, { toValue: { x: 0, y: 0 }, useNativeDriver: false }).start();
         }
       }
     });
